### PR TITLE
Create ecological-restoration.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/67745/style-request-ecological-restoration

### CODE REVIEW

File: ecological-restoration.csl [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+          </else>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="locators">
+    <choose>
+      <if type="article-journal article-magazine article-newspaper" match="any">
+        <choose>
+          <if match="all" variable="page issued">
```
Comment (adam3smith) [inline ecological-restoration.csl]: Are "page issued" really the best variables here? Wouldn't "page volume" make more sense here?

File: ecological-restoration.csl [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+    </sort>
+    <layout prefix="(" suffix=")" delimiter=", ">
+      <group delimiter=" ">
+        <text macro="author-short"/>
+        <text macro="issued-year"/>
+        <text macro="citation-locator"/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography entry-spacing="0" hanging-indent="true" delimiter-precedes-last="contextual">
```
Comment (adam3smith) [inline ecological-restoration.csl]: remove the delimiter-precedes-last here: it doesn't do anything (as this is the default) and this isn't a good place to set it, either.

File: ecological-restoration.csl [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+  <macro name="locators">
+    <choose>
+      <if type="article-journal article-magazine article-newspaper" match="any">
+        <choose>
+          <if match="all" variable="page issued">
+            <text variable="container-title" prefix=". " font-style="italic"/>
+            <text variable="volume" prefix=" "/>
+            <text variable="page" prefix=":"/>
+          </if>
+          <else>
```
Comment (adam3smith) [inline ecological-restoration.csl]: You almost certainly would still want the container-title for early publication items, right?

**Comment by adam3smith** [comment]
A couple of small comments/questions above (sorry for the multiple notifications, I thought I just had one & wasn't going to use review mode but then spotted a couple of other potential issues).

**Comment by rmzelle** [comment]
(metadata looks good to me)

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**